api/nakadi-event-bus-api.yaml

```diff
@@ -723,39 +736,138 @@ definitions:
   Event:
     type: object
     description: |
-      **To be changed**. This is the most general representation of an event, that can be processed
-      by Nakadi.
+      **Note** The Event definition will be externalized in future versions of this document.
+
+      A basic payload of an Event. The actual schema is dependent on the information configured for the
+      EventType, as is its enforcement (see POST /event-types). Setting of metadata properties are
+      dependent on the configured enrichment as well.
+
+      For explanation on default configurations of validation and enrichment, see documentatin of
+      `EventType.type`.
+
+      For concrete examples of what will be enforced by Nakadi see the objects BusinessEvent and
+      DataChangeEvent below.
+
 
-      It should be used as a base definition for all events, that flow through
-      Nakadi by extending attributes of this object type.
+  EventMetadata:
+    type: object
+    description: |
+      fooooog
     properties:
-      event_type:
-        type: string
-        example: 'https://resource-events.zalando.com/ResourceCreated'
-      partitioning_key:
-        type: string
-        example: 'ABC123XXX-001'
       metadata:
-        $ref: '#/definitions/EventMetaData'
-  EventMetaData:
+        type: object
+        properties:
+          eid:
+            description: |
+              Identifier of this Event.
+
+              Clients are allowed to generate this and this SHOULD be guaranteed to be unique from the
+              perspective of the producer. Consumers MIGHT use this value to assert uniqueness of reception
+              of the Event.
+
+              If not present upon reception, Nakadi will generate and fill this with a unique value.
+              (TBD: enforce UUID formatif set by the client?)
+            type: string
+            format: uuid
+            example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
+          event-type:
+            description: |
+              The EventType of this Event. This is enriched by Nakadi on reception of the Event based on the
+              endpoint where the Producer sent the Event to.
+
+              If provided MUST match the endpoint. Failure to do so will cause rejection of the Event.
+            type: string
+            example: 'pennybags:payment-business-event'
+          ocurred-at:
```

Comment:
filed names for the entities should be snale_case, @jkliff you wanted to rename these, but did not find time yet